vips/vip-350/vip-350/commands.ts

```diff
@@ -0,0 +1,44 @@
+import { Assets, BaseAssets } from "./addresses";
+
+type IncentiveAndAccessibility = [number, number];
+
+function getIncentiveAndAccessibility(baseAsset: string) {
+  const incentivesAndAccess: Array<[number, number]> = [];
+
+  for (const asset of Assets) {
+    if (asset != baseAsset) {
+      // Every conversion of the baseAsset for USDT in a SingleTokenConverter with a baseAsset != USDT is enabled only for converters,
+      // because the RiskFundConverter of the USDTPrimeConverter should be able to cover those conversions
+      if (asset === BaseAssets[0]) {
+        if (baseAsset != BaseAssets[5]) {
+          incentivesAndAccess.push([0, 2]); // ONLY_FOR_CONVERTERS
+        } else {
+          incentivesAndAccess.push([0, 1]); // ALL for XVSVaultConverter
```

Comment:
Why are we not configuring an incentive for USDT/XVS conversion in the XVSVaultConverter? I think we can also remove this command

Reply:
https://github.com/VenusProtocol/vips/pull/339#discussion_r1711727536